src/applications/debt-letters/app-entry.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import 'platform/polyfills';
+import './sass/debt-letters.scss';
+import DebtLettersWrapper from './components/DebtLettersWrapper';
+
+import startApp from '../../platform/startup';
```

Comment:
Please fix this and use the alias instead
```suggestion
import startApp from 'platform/startup';
```